pgmq-extension/sql/pgmq--1.3.3--1.3.4.sql

```diff
@@ -0,0 +1,66 @@
+-- Add function to convert archive queues to partitioned tables
+CREATE FUNCTION pgmq.convert_archive_partitioned(table_name TEXT,
+                                                 partition_interval TEXT DEFAULT '10000',
+                                                 retention_interval TEXT DEFAULT '100000',
+                                                 leading_partition INT DEFAULT 10)
+RETURNS void AS $$
+DECLARE
+a_table_name TEXT;
+a_table_name_old TEXT;
+qualified_a_table_name TEXT;
+qualified_a_table_name_old TEXT;
+BEGIN
+  PERFORM pgmq._ensure_pg_partman_installed();
+  SELECT FORMAT('%s', 'a_'|| table_name) INTO a_table_name;
+  SELECT FORMAT('%s', 'a_'|| table_name || '_old') INTO a_table_name_old;
+  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name) INTO qualified_a_table_name;
+  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name || '_old') INTO qualified_a_table_name_old;
```

Comment:
this section is different than how it is defined in `pgmq.sql`. These need to be the same.

Reply:
Addressed. Thanks!